scss/_patterns_navigation.scss

```diff
@@ -245,7 +245,7 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
   .p-navigation__tagged-logo {
     display: flex; // to prevent logo link from expanding full width
     margin-right: 0;
-    min-width: 13rem;
+    min-width: 7rem;
```

Comment:
There was a reason this was set to 13rem. At least on desktop it should be like that.

If we need to reduce it (on smaller screens), we should still keep it at original size on desktop.

Reply:
Reverted

Reply:
@ClementChaumel Could you address that? I think this can be removed here on smaller screens, if not let me know, but we should not assume the width of logo this way.